css/service.css

```diff
@@ -0,0 +1,297 @@
+    align-items: baseline; 
+}
+
+nav {
+    display:flex;
+    flex-wrap: wrap;
+}
+
+
+a {
+    padding: 25px;
+    color:gray;
+    text-decoration: none;
+}
+
+
+
+.image-top {
+    margin-top: 200px;
+    margin-bottom: 50px;
+    text-align: center;
+    max-width: 100%;
+    
+}
+
+.section-services {
+    margin-top:30px;
+}
+
+
+.section-one {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 40px 0px;
+    border-top: 2px solid black;
+    padding-top: 40px;
+}
+
+.section-design, .section-ui {
+    max-width: 470px;
+    height: 320px;
+    border: 1px solid black;
+    padding:30px;
+    background:#f0efef;
+    opacity:0.8;
+}
+.section-design:hover, .section-ui:hover {
+    opacity:2;
+    color: purple;
+    -webkit-transform: rotateZ(-05deg);
+    -ms-transform: rotateZ(-05deg);
+    transform: rotateZ(-05deg);
+}
+
+.section-two {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 40px 0px;
+}
+
+.section-digital, .section-web {
+    max-width: 470px;
+    height: 320px;
+    border: 1px solid black;
+    padding:30px;
+    background:#f0efef;
+    
+}
+
+.section-digital:hover, .section-web:hover {
+    border-radius:25%;
+}
+
+
+
+.section-three {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    margin: 40px 0px;
+}
+
+.section-ios, .section-android {
+    max-width: 470px;
+    height: 320px;
+    border: 1px solid black;
+    padding:30px;
+    background:#f0efef;
+}
+
+
+.section-footer {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-between;
+    border-top: 2px solid black;
+    padding-top: 40px;
+}
+
+.section-funfacts {
+    max-width: 500px;
+    height: 300px;
+}
+
+.footer-list {
+    margin-left: 20px;
+    margin-top: 50px;
+}
+
+.copyright {
+    margin: 60px 100px;
+    text-align: center;
+    font-family: Brush Script MT;
+}
+button {
+    padding: 10px;
+    border-radius: 10px;
+    margin-top: 60px;
+    width: 130px;
+}
+
+button:hover{
+    color:pink;
+}
+
+section p {
+    margin-top:40px;
+}
+
+img{
+    max-width:100%;
+    height: auto;
+}
+
+
+@media (max-width:80rem) {
+
+    .container{
+        width: 90%;
+        border:1px solid black;
+        display:flex;
+        align-items: center;
+    }
+
+    header{
+        background:white;
+        display:flex;
+        flex-direction: column-reverse;
+        align-items: center;
+    }
+
+    header nav {
+        display: flex;
+        justify-content: space-evenly;
+    }
+    .services-header-img{
+        display:none;
+    }
+
+    .section-services {
+        padding:20px;
+    }
+
+    .section-one{
+        display:flex;
+        flex-direction: column;
+    }
+
+    .section-two {
+        flex-direction: column;
+    }
+
+    .section-three {
+        flex-direction: column;
+    }
+
+    .section-footer {
+        flex-direction: column;
+    } 
+}
+
+@media (max-width:50rem){
```

Comment:
Yet again i would go with pixels. Pixels are the only unit that behave consistently across all commonly used browsers.